v5/extended/apix.rst

```diff
@@ -33,12 +33,12 @@ Control settings of the way the file's driver treats the file.
 registry_bind_port
 ------------------
 
-Registers a device of the given type in the given port into the registry, if
+Registers a device of the given type in the given zero-indexed port into the registry, if
 that type of device is detected to be plugged in to that port.
 
 This function uses the following values of ``errno`` when an error state is reached:
 
-- ``EINVAL``     - The given value is not within the range of V5 ports (1-21).
+- ``EINVAL``     - The given value is not within the range of V5 ports (0-20).

```

Comment:
Isn't every other port 1 indexed? (from a Glace that is what it looks like.) why is this different?

Reply:
This change was made a to keep the iternal API consistent... turns out some of the functions were also documented incorrectly and it was decided that we keep them 0 indexed.